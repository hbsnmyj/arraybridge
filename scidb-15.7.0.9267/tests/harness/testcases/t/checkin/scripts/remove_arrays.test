--setup
create array fdafdarfewqfveqfeq<v:int64>[i]
create array fdafdarfewqfveqfeq2<v:int64>[i]

--test
--start-query-logging
project(filter(list(), name='fdafdarfewqfveqfeq' or name='fdafdarfewqfveqfeq2'), name)
--shell --command "${SCIDB_INSTALL_PATH}/bin/remove_arrays.py -f fdafdarfewqfveqfeq.*"
project(filter(list(), name='fdafdarfewqfveqfeq' or name='fdafdarfewqfveqfeq2'), name)
--stop-query-logging

--cleanup

