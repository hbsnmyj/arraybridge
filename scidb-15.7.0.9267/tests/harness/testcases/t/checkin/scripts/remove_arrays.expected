Query was executed successfully

Query was executed successfully

SCIDB QUERY : <project(filter(list(), name='fdafdarfewqfveqfeq' or name='fdafdarfewqfveqfeq2'), name)>
{No} name
{0} 'fdafdarfewqfveqfeq'
{1} 'fdafdarfewqfveqfeq2'

SCIDB QUERY : <project(filter(list(), name='fdafdarfewqfveqfeq' or name='fdafdarfewqfveqfeq2'), name)>
{No} name

